packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/new-account-form-controller.js

```diff
@@ -407,7 +407,7 @@ export default class NewAccountFormController {
           this.originalManagerLanguage !== this.model.managerLanguage
         ) {
           this.coreConfig.setUserLocale(this.model.managerLanguage);
-          window.location.reload();
+          window.parent.location.reload();
```

Comment:
We need to add an utility method in the container/shell to make the reload. 
Current implementation will not be appropriate if we run the dedicated app without container.

Reply:
Hello Anoop,
I did this because "window.parent" returns the same window object if you are in an iFrame or not. So it will work the same, with or without the container.